config/config.ts

```diff
@@ -0,0 +1,15 @@
+import * as fs from 'fs'
+
+let configFile
+let localConfig
+
+// If project is being run locally, the developer will have their API key in the json file
+try {
+  configFile = fs.readFileSync('./config/hollowbot.config.json', 'utf8')
```

Comment:
> Sorry about not being clear earlier. I was thinking hollowbot.config.json would be the default config file that lives outside of HollowBot's repo (and HollowBot would know where to look for it).
Just to clarify: according to the last status of the repo `hollowbot.config.json` is living outside of the repository, is it not?